BPA/Ch.13/owls_ms1.stan

```diff
@@ -18,27 +18,27 @@ transformed parameters {
   simplex[3] phi[R];            // State vector
   simplex[3] p[3, T];           // Observation matrix
 
-  p3 <- beta / sum(beta);       // Induce Dirichlet prior
+  p3 = beta / sum(beta);       // Induce Dirichlet prior
 
   // Define state vector
   for (s in 1:R) {
-    phi[s, 1] <- 1.0 - psi;          // Prob. of non-occupation
-    phi[s, 2] <- psi * (1.0 - r);    // Prob. of occupancy without repro
-    phi[s, 3] <- psi * r;            // Prob. of occupancy and repro
+    phi[s, 1] = 1 - psi;          // Prob. of non-occupation
+    phi[s, 2] = psi * (1 - r);    // Prob. of occupancy without repro
+    phi[s, 3] = psi * r;            // Prob. of occupancy and repro
   }
 
   // Define observation matrix
   // Order of indices: true state, time, observed state
   for (t in 1:T) {
-    p[1, t, 1] <- 1.0;
-    p[1, t, 2] <- 0.0;
-    p[1, t, 3] <- 0.0;
-    p[2, t, 1] <- 1.0 - p2;
-    p[2, t, 2] <- p2;
-    p[2, t, 3] <- 0.0;
-    p[3, t, 1] <- p3[1];
-    p[3, t, 2] <- p3[2];
-    p[3, t, 3] <- p3[3];
+    p[1, t, 1] = 1;
```

Comment:
Your way may be clearer, but you could also do this via:

```
p[1, 1:T, 1] <- rep_array(1, T);
p[1, 1:T, 2] <- rep_array(0, T);
```
and so on.  I'm not 100% sure this'll work, but it should.

It's more efficient this way as you do less arithmetic copying than continually recalculating `1 - p2` and creating distinct zero variables that aren't tied back to any parameters (so can all be shared).